ts/features/walletV3/payment/screens/WalletPaymentOutcomeScreen.tsx

```diff
@@ -204,6 +208,18 @@ const WalletPaymentOutcomeScreen = () => {
     }
   };
 
+  const WaitingConfirmationEmailSubtitle = () => (
+    <>
+      <LabelSmall style={styles.text} color="grey-650" weight="Regular">
+        {I18n.t("wallet.payment.outcome.WAITING_CONFIRMATION_EMAIL.subtitle")}{" "}
+        <LabelSmall style={styles.text} color="grey-650" weight="SemiBold">
+          {profileEmail}
+        </LabelSmall>
```

Comment:
While this is an edge case, are we sure that we want to display a possibly undefined value here?  I'm imagining the case where we get a copy along the lines of "you will get a receipt in this email adress: ", maybe an alternative copy to handle the undef case, like "you will receive a receipt in your email address." could be a good idea, what do you think?